utils/wasm-builder/src/wasm_project.rs

```diff
@@ -335,6 +246,33 @@ fn create_wasm_workspace_project(wasm_workspace: &Path, workspace_root_path: &Pa
 		wasm_workspace_toml.insert("patch".into(), patch.into());
 	}
 
+	let mut package = Table::new();
+	package.insert("name".into(), format!("{}-wasm", crate_name).into());
+	package.insert("version".into(), "1.0.0".into());
+	package.insert("edition".into(), "2018".into());
+
+	wasm_workspace_toml.insert("package".into(), package.into());
+
+	let mut lib = Table::new();
+	lib.insert("name".into(), wasm_binary.into());
+	lib.insert("crate-type".into(), vec!["cdylib".to_string()].into());
+
+	wasm_workspace_toml.insert("lib".into(), lib.into());
+
+	let mut dependencies = Table::new();
+
+	let mut wasm_project = Table::new();
+	wasm_project.insert("package".into(), crate_name.into());
+	wasm_project.insert("path".into(), crate_path.display().to_string().into());
+	wasm_project.insert("default-features".into(), false.into());
+	wasm_project.insert("features".into(), enabled_features.to_vec().into());
+
+	dependencies.insert("wasm-project".into(), wasm_project.into());
+
+	wasm_workspace_toml.insert("dependencies".into(), dependencies.into());
+
+	wasm_workspace_toml.insert("workspace".into(), Table::new().into());
```

Comment:
Yes, otherwise cargo will complain that the project is in a workspace, but not registered in this workspace. (The workspace is the one of the original project)